pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Messages.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.client.api;
+
+import java.util.List;
+
+/**
+ * A container that holds the list {@link Message} for a topic.
+ * @param <T>
+ */
+public interface Messages<T> extends Iterable<Message<T>> {
+
+    /**
+     * Get the list {@link Message}
+     */
+    List<Message<T>> getMessageList();
```

Comment:
do we need to expose this? my feeling is to avoid exposing such method until it is really needed.

Reply:
remove it